src/config/dicom/tagConversion.ts

```diff
@@ -0,0 +1,50 @@
+import * as dcmjs from 'dcmjs'
+
+/**
+ * Check if a tag identifier is a private tag
+ * Private tags have odd group numbers (e.g., 0005, 0007, 0009, etc.)
+ */
+export function isPrivateTag(tagId: string): boolean {
```

Comment:
A few ideas:
* centralize this functionality by reusing the dcmjs Tag class and extending it as needed
* https://github.com/dcmjs-org/dcmjs/blob/master/src/Tag.js
* Use Tag instead to TagId for consistency